docs/guides/Package-authoring-best-practices.md

```diff
@@ -0,0 +1,191 @@
+
+### Package Version
+
+✔️ CONSIDER using [SemVer](https://semver.org/) to version your NuGet package.
+> Essentially, this means using the Major.Minor.Patch[-prerelease] format.
+
+✔️ DO include a pre-release suffix when releasing a non-stable package.
+TODO: Include and example and link to https://docs.microsoft.com/dotnet/standard/library-guidance/nuget#pre-release-packages somewhere
+
+See the [.NET library versioning guide](https://docs.microsoft.com/dotnet/standard/library-guidance/versioning) for more advanced guidance including assembly versioning.
+
+### Authors
+
+✔️ CONSIDER including the current NuGet.org owners for consistency across NuGet clients.
+> Visual Studio currently displayed `Authors` on the Browse tab. It is also in the criteria for ID prefix reservation.
+
+### Description
+
+✔️ DO include a short description for your package that describes what it's for.
+> Package descriptions are one of the most prominent fields surfaced in NuGet search and will likely be the first thing a potential consumers looks at to determine if a package is right for them.
+
+### Copyright
+
+✔️ CONSIDER copyrighting your package with "Copyright (c) <name/company\> <year\>."
+>A copyright notice essentially indicates that your work cannot be copied without your permission. Including a copyright notice in your package is easy and won't do any harm!
+
+Example: Copyright (c) Contoso 2020
+
+### Licensing
+
+✔️ DO [include a valid license expression or license file in your package](https://docs.microsoft.com/nuget/reference/msbuild-targets#packing-a-license-expression-or-a-license-file).
+> [!IMPORTANT]
+> A project without a license defaults to [exclusive copyright](https://choosealicense.com/no-permission/), making it legally impossible for other people to use.
+
+❌ DO NOT use the deprecated `LicenseUrl` metadata property.
+> This presents legal ambiguity as license changes at the URL will retroactively change the displayed license for previous package versions.
+
+#### If your package is [open source](https://opensource.org/osd)
+
+✔️ DO [choose an open source license](https://choosealicense.com/) to make your package open source.
+> *"Open source licenses are licenses that comply with the Open Source Definition — in brief, they allow software to be freely used, modified, and shared."* - Open Source Initiative. To learn more about open source software and the Open Source Initiative, check out https://opensource.org/.
+
+✔️ CONSIDER [including a license expression in your package](https://docs.microsoft.com/nuget/reference/msbuild-targets#packing-a-license-expression-or-a-license-file).
+> License expressions are surfaced the most clearly and make it more obvious to consumers if they can use your package or if the license has changed. 
+> [!Note]
+> NuGet.org only accepts license expressions that are approved by the Open Source Initiative or the Free Software Foundation.
```

Comment:
Consider changing to:

> NuGet.org only accepts license expressions with licenses that are approved by the Open Source Initiative or the Free Software Foundation.

Because license expressions aren’t approved by OSI or FSF, only licenses.

> NuGet.org only accepts   license expressions with licenses that are approved by the Open Source   Initiative or the Free Software Foundation.
--